File: app/src/main/java/br/com/caiopo/travelbuddy/model/api/ConversionApi.java
package br.com.caiopo.travelbuddy.model.api;

import br.com.caiopo.travelbuddy.model.entity.Conversion;
import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Path;

public interface ConversionApi {

    @GET("/convert?q={query}")
    Call<Conversion> getConversion(@Path("query") String query);
}

File: app/src/main/java/br/com/caiopo/travelbuddy/model/api/conversion/ConversionApi.kt
package br.com.caiopo.travelbuddy.model.api.conversion

import br.com.caiopo.travelbuddy.model.entity.Conversion
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface ConversionApi {

    @GET("convert")
    fun getConversion(@Query("q") query: String): Call<Conversion>
}
